types/src/v0/impls/block/full_payload/ns_proof/advz.rs

```diff
@@ -0,0 +1,328 @@
+use hotshot_types::{
```

Comment:
It would be a bit easier to review if it was mentioned that this file contains the previous `ns_proof.rs`

Reply:
added new description here: [67d6095](https://github.com/EspressoSystems/espresso-network/pull/2866/commits/67d6095c3f74bb6a3930790a8bc4e60e5e5bcd50)